SPECS/libsoup/libsoup.spec

```diff
@@ -61,22 +65,23 @@ Requires:       %{name} = %{version}-%{release}
 These are the additional language files of libsoup.
 
 %prep
-%setup -q
-%patch0 -p1
+%autosetup
 
 %build
-%configure --disable-vala --enable-introspection=yes
-make %{?_smp_mflags}
+%meson \
+    -Dgtk_doc=true \
+    -Dsysprof=disabled \
+    -Dautobahn=disabled \
+    -Dhttp2_tests=disabled \
+    -Dntlm=disabled \
+    -Dtests=false
+%meson_build
 
 %install
-rm -rf %{buildroot}%{_infodir}
-make DESTDIR=%{buildroot} install
+%meson_install
 find %{buildroot} -type f -name "*.la" -delete -print
 
-%find_lang %{name}
-
-%check
```

Comment:
Why did we need to remove the `%check` section?

Reply:
Added back